Scripts/InitializeEnvironment.bat

```diff
@@ -3,6 +3,7 @@
 REM Set environment variables for interesting paths that scripts might need access to.
 PUSHD %~dp0
 SET VFS_SCRIPTSDIR=%CD%
+POPD

```

Comment:
This is just cleanup and not related to your change right? (I didn't know Windows had pushd/popd so I learned something new!)

Reply:
Yes I'm just fixing a bug in this script. I hit this because when I tried running the prefetch functional tests, it kept changing my current directory.